src/inference/src/ie_core.cpp

```diff
@@ -740,6 +749,11 @@ class CoreImpl : public ie::ICore, public std::enable_shared_from_this<ie::ICore
                 if (disabled)
                     return;
             }
+            if (!coreConfig.flag_allow_auto_batching) {
+                const auto disabled = !coreConfig.flag_allow_auto_batching;
+                if (disabled)
```

Comment:
1. "if" has checked "!coreConfig.flag_allow_auto_batching" is true, we don't need to above redundant code.
2. in line 741, we can get the value of ov::hint::allow_auto_batching, why apply a new flag - coreConfig.flag_allow_auto_batching - to check it again?

Reply:
1.I will simplify it.
2.When we set properties, in openvino/src/plugins/intel_cpu/src/config.cpp, the void Config::readProperties function will read properties. But the ov::hint::allow_auto_batching is not recognized in this function. So I create a new variable to store the value of ov::hint::allow_auto_batching and release the keyword from the config map variable.